probe-rs/src/architecture/arm/sequences.rs

```diff
@@ -919,9 +977,13 @@ pub trait DebugEraseSequence: Send + Sync {
 }
 
 /// Perform a SWD line reset (SWDIO high for 50 clock cycles)
-fn swd_line_reset(interface: &mut dyn DapProbe) -> Result<(), ArmError> {
+///
+/// After the line reset, SWDIO will be kept low for `swdio_low_cycles` cycles.
+fn swd_line_reset(interface: &mut dyn DapProbe, swdio_low_cycles: u8) -> Result<(), ArmError> {
+    assert!(swdio_low_cycles + 51 <= 64);
+
     tracing::debug!("Performing SWD line reset");
-    interface.swj_sequence(51, 0x0007_FFFF_FFFF_FFFF)?;
+    interface.swj_sequence(51 + swdio_low_cycles, 0x0007_FFFF_FFFF_FFFF)?;
```

Comment:
It's high for 51 cycles, and then low for additional cycles. The reason behind combining it is just to avoid having to do two calls to `swj_sequence`.